### PR TITLE
NullReferenceException in PropertyGrid in Debug Mode

### PR DESCRIPTION
**Pull request by aldinei-sampaio**
Correction for issues #39 and #46.

### CODE REVIEW

**Comment by objorke** [comment]
Thanks for contributing!

File: Source/PropertyTools.Wpf/PropertyGrid/PropertyGrid.cs [hunk 1]
```diff
@@ -979,7 +979,7 @@ public virtual void CreateControls(object instance, IEnumerable<Tab> tabs)
                     Grid.SetIsSharedSizeScope(tabPanel, true);
                 }
 
-                var tabItem = new TabItem { Header = tab, Padding = new Thickness(4), Name = tab.Id };
+                var tabItem = new TabItem { Header = tab, Padding = new Thickness(4), Name = (tab.Id == null ? String.Empty : tab.Id) };
```
Comment (objorke) [inline Source/PropertyTools.Wpf/PropertyGrid/PropertyGrid.cs]: a detail: wouldn't it be better to use `Name = tab.Id ?? String.Empty`?

Comment (aldinei-sampaio) [inline Source/PropertyTools.Wpf/PropertyGrid/PropertyGrid.cs]: Sure looks better. I am not fully used to C# yet, and in VB.NET we dont have the ?? operator.

**Comment by objorke** [comment]
Will you create a new PR? It is possible to update this PR with new commits (commit the change, then do an interactive rebase to squash the commits)

**Comment by aldinei-sampaio** [comment]
I will try. I am quite newbie with Github stuff yet. =P

**Comment by aldinei-sampaio** [comment]
Issue #57 oppenned.

**Pull request closed without merging**